installer/tue-install-impl.bash

```diff
@@ -1222,6 +1222,7 @@ shift
 
 # idiomatic parameter and option handling in sh
 targets=""
+BRANCH=
 while test $# -gt 0
```

Comment:
More consistent to use `BRANCH=""`?

Reply:
I just kept it same as other parameters defined below the parameter handling loop. Can use your suggestion if you like for this line

Reply:
I just like some consistentcy. I don't have a preference for one or the other. But it conflicts with the targets line above it.

Reply:
Haha your OCD is not allowing you to go past that line :rofl: 

I'll change it.